WebApplicationTests/PatientRegistrationTest.cs

```diff
@@ -0,0 +1,97 @@
+            stubRepository.Setup(m => m.addPatient(patient)).Returns(true);
+            RegistrationService service = new RegistrationService(stubRepository.Object);
+
+            bool addedPatient = service.RegisterPatient(patient);
+
+            Assert.True(addedPatient);
+        }
+
+
+        [Fact]
+        public void JmbgIsValidTest()
+        {
+            var stubRepository = new Mock<IRegistrationRepository>();
+            IRegistrationRepository r = new RegistrationRepository();
+            Patient p = new Patient("2", "Ana", "Anic", "1234", DateTime.Now, "0643342345", "ana@gmail.com", new Address { Street = "Glavna 3" }, "Jovan", "Beograd", "Savski venac", "Srbija", "Srpsko", "Srbin", "Doktor", "Ruma", "Ruma", "Srbija", "employed", "merried", "223345677", "", "", "female", "ana123", "image", false);
+            stubRepository.Setup(m => m.GetPatientIdById("123")).Returns(p.Id);
+            RegistrationService service = new RegistrationService(stubRepository.Object);
+
+            bool hasPatient = service.IsJMBGValid("1234");
+
+            Assert.True(hasPatient);
+        }
+
+        [Fact]
+        public void JmbgIsNotValidTest()
+        {
+            var stubRepository = new Mock<IRegistrationRepository>();
+            IRegistrationRepository r = new RegistrationRepository();
+            Patient p = new Patient("2", "Ana", "Anic", "1234", DateTime.Now, "0643342345", "ana@gmail.com", new Address { Street="Glavna 3"}, "Jovan", "Beograd", "Savski venac", "Srbija", "Srpsko", "Srbin", "Doktor", "Ruma", "Ruma", "Srbija", "employed", "merried", "223345677", "", "", "female", "ana123", "image", false);
+            stubRepository.Setup(m => m.GetPatientIdById("1234")).Returns(p.Id);
+            RegistrationService service = new RegistrationService(stubRepository.Object);
+
+            bool hasPatient = service.IsJMBGValid("1234");
+
+            Assert.False(hasPatient);
+        }
+
+        [Fact]
+        public void ConfirmRegistrationTest()
+        {
+            var mockRepository = new Mock<IRegistrationRepository>();
+            Patient patient = new Patient("2", "Ana", "Anic", "1234", DateTime.Now, "0643342345", "ana@gmail.com", new Address { Street = "Glavna 3" }, "Jovan", "Beograd", "Savski venac", "Srbija", "Srpsko", "Srbin", "Doktor", "Ruma", "Ruma", "Srbija", "employed", "merried", "223345677", "", "", "female", "ana123", "image", false);
+            mockRepository.Setup(m => m.ConfirmgEmailUpdate(patient.Id)).Returns(true);
+            RegistrationService service = new RegistrationService(mockRepository.Object);
+
+            bool patientUpdated = service.ConfirmgEmailUpdate(patient.Id);
+
+            Assert.True(patientUpdated);
+
+        }
+
+        [Fact]
+        public void SendingMailTest()
```

Comment:
add empty lines between assign, act and assert parts to keep up the good convention you've set up

Reply:
function name